src/helper/post_message_helper.ts

```diff
@@ -13,19 +13,20 @@ const PostMessageHelper = async (
 	context: Context,
 	callback: Callback
 ) => {
-	const configurator = new Configurator()
-	const token = configurator.getSlackConfig().token
+	const configurator = new Configurator("env.yml")
+	const authToken = configurator.getSlackConfig().authToken
 	const channel = configurator.getSlackConfig().channel
 	const channelName = configurator.getSlackConfig().channelName
-	const client = new SlackClient(token)
+	const client = new SlackClient(authToken)
```

Comment:
we should have two SlackClient, 
One SlackClient to retrieve channel info , 
Another SlackClient with both authentication token to open conversation and send a message